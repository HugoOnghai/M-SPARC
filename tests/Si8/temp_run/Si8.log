
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat Si8.inpt
# nprocs: 24
# Test: Si8 #
LATVEC:
1.000000000000000 0.000000000000000 0.000000000000000
-0.292371704722737 0.956304755963035 0.000000000000000
0.173648177666930 -0.110492654830881 0.978589640054184
CELL: 10.2 10.2 10.2
FD_GRID: 25 25 25
FD_ORDER: 12
BC: P P P
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.773
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
#ELEC_TEMP: 
NSTATES: 30
CALC_PRES: 1
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none

# MD
#MD_FLAG:  1
#MD_METHOD: NVT_NH
#MD_TIMESTEP: 1
#MD_NSTEP: 10
#PRINT_RESTART_FQ: 3
#RESTART_FLAG: 0

<\INPT>

 Reading .ion file ...
 Number of atom types : 1
 Total number of atoms: 8
 Default atomic mass for Si is 28.085000
 pseudo_dir # 1 = Si.psp8

 COORD:
                   0                   0                   0
   5.100000000000000   5.100000000000000                   0
                   0   5.100000000000000   5.100000000000000
   5.100000000000000                   0   5.100000000000000
   2.550000000000000   2.550000000000000   2.550000000000000
   7.649999999999999   7.649999999999999   2.550000000000000
   2.550000000000000   7.649999999999999   7.649999999999999
   7.649999999999999   2.550000000000000   7.649999999999999

   0.030168927102176

 kpoint grid before symmetry:
     0     0     0

 kpoint grid after symmetry:
     0     0     0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 20
 Creating differentiation matrices ...
 Done. (3.401 sec)

 Estimated memory usage:
 Total:   57.94 MB
 orbitals            :   14.31 MB
 sparse matrices     :   40.29 MB
 global-size vectors :    1.67 MB
 mixing histories    :    1.67 MB

 Finding rb for Si ...
 rb = {4.080 4.080 4.080}, int_b = -4.000016737766446, err_rb = 1.674e-05
 rb = {4.488 4.488 4.488}, int_b = -4.000014873694367, err_rb = 1.487e-05
 rb = {4.896 4.896 4.896}, int_b = -4.000013006541776, err_rb = 1.301e-05
 rb = {5.304 5.304 5.304}, int_b = -4.000011313744060, err_rb = 1.131e-05
 rb = {5.712 5.712 5.712}, int_b = -4.000010600093008, err_rb = 1.060e-05
 rb = {6.120 6.120 6.120}, int_b = -4.000002144545152, err_rb = 2.145e-06
 rb = {6.528 6.528 6.528}, int_b = -3.999999792583405, err_rb = 2.074e-07
 rb = {6.936 6.936 6.936}, int_b = -4.000000045355511, err_rb = 4.536e-08
 rb = {7.344 7.344 7.344}, int_b = -3.999999988932631, err_rb = 1.107e-08
 rb = {7.752 7.752 7.752}, int_b = -4.000000001945953, err_rb = 1.946e-09
 rb = {7.752 7.752 7.752}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 32.000000021287

 Integration b_ref = 32.000000000010

 ****************************************
 *          Eself_ref = 225.410067       *
 ****************************************
 Done. (2.371207 s)
 Time for b calculation: 2.385 seconds.

 Starting calculating nonlocal projectors ...
 Done. (1.491540 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 9.88891e-09 in 29 iterations.
 Poisson problem took 0.771736s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 1.661026
 This SCF iteration took 3.327 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.370450
 This SCF iteration took 1.653 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = 0.239417
 This SCF iteration took 1.455 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = 0.229595
 AAR converged to a relative residual of 6.96999e-09 in 25 iterations.
 Poisson problem took 0.653347s
 Escc = 0.44870914
 Etot = -31.30908976
 Eatom = -3.91363622
 Error in SCF iteration: 2.3087e-02 
 This SCF iteration took 2.153 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.235474
 AAR converged to a relative residual of 2.99257e-09 in 25 iterations.
 Poisson problem took 0.654085s
 Escc = 0.07271801
 Etot = -31.32181085
 Eatom = -3.91522636
 Error in SCF iteration: 1.4311e-02 
 This SCF iteration took 2.153 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.234379
 AAR converged to a relative residual of 1.64206e-09 in 25 iterations.
 Poisson problem took 0.643954s
 Escc = 0.04651594
 Etot = -31.32364566
 Eatom = -3.91545571
 Error in SCF iteration: 8.5126e-03 
 This SCF iteration took 2.222 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.233918
 AAR converged to a relative residual of 9.99346e-10 in 25 iterations.
 Poisson problem took 0.650856s
 Escc = -0.00440398
 Etot = -31.32429812
 Eatom = -3.91553726
 Error in SCF iteration: 4.2169e-03 
 This SCF iteration took 2.176 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.233587
 AAR converged to a relative residual of 9.67445e-09 in 21 iterations.
 Poisson problem took 0.548996s
 Escc = -0.00880742
 Etot = -31.32440925
 Eatom = -3.91555116
 Error in SCF iteration: 2.1866e-03 
 This SCF iteration took 2.034 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = 0.233533
 AAR converged to a relative residual of 8.91147e-09 in 19 iterations.
 Poisson problem took 0.490494s
 Escc = -0.00590269
 Etot = -31.32443894
 Eatom = -3.91555487
 Error in SCF iteration: 8.9909e-04 
 This SCF iteration took 1.962 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = 0.233544
 AAR converged to a relative residual of 2.52862e-09 in 19 iterations.
 Poisson problem took 0.493453s
 Escc = -0.00248501
 Etot = -31.32444276
 Eatom = -3.91555534
 Error in SCF iteration: 4.3896e-04 
 This SCF iteration took 1.935 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = 0.233560
 AAR converged to a relative residual of 9.7079e-10 in 19 iterations.
 Poisson problem took 0.496813s
 Escc = -0.00080023
 Etot = -31.32444391
 Eatom = -3.91555549
 Error in SCF iteration: 1.8939e-04 
 This SCF iteration took 1.958 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = 0.233564
 AAR converged to a relative residual of 9.23293e-09 in 15 iterations.
 Poisson problem took 0.390752s
 Escc = -0.00005334
 Etot = -31.32444432
 Eatom = -3.91555554
 Error in SCF iteration: 6.9151e-05 
 This SCF iteration took 1.849 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = 0.233563
 AAR converged to a relative residual of 6.76317e-09 in 13 iterations.
 Poisson problem took 0.335557s
 Escc = 0.00000637
 Etot = -31.32444432
 Eatom = -3.91555554
 Error in SCF iteration: 2.5687e-05 
 This SCF iteration took 1.784 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = 0.233560
 AAR converged to a relative residual of 1.23187e-09 in 13 iterations.
 Poisson problem took 0.340144s
 Escc = -0.00001039
 Etot = -31.32444423
 Eatom = -3.91555553
 Error in SCF iteration: 1.3752e-05 
 This SCF iteration took 1.798 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = 0.233558
 AAR converged to a relative residual of 8.3629e-10 in 13 iterations.
 Poisson problem took 0.339637s
 Escc = 0.00000960
 Etot = -31.32444422
 Eatom = -3.91555553
 Error in SCF iteration: 4.3132e-06 
 This SCF iteration took 1.806 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = 0.233558
 AAR converged to a relative residual of 9.05185e-09 in 8 iterations.
 Poisson problem took 0.204422s
 Escc = 0.00000367
 Etot = -31.32444422
 Eatom = -3.91555553
 Error in SCF iteration: 2.0351e-06 
 This SCF iteration took 1.651 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = 0.233558
 AAR converged to a relative residual of 7.80608e-09 in 7 iterations.
 Poisson problem took 0.182241s
 Escc = 0.00000133
 Etot = -31.32444420
 Eatom = -3.91555552
 Error in SCF iteration: 7.9657e-07 
 This SCF iteration took 1.632 s.


 Finished SCF iteration in 14 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -31.324444199 Ha.       *
 *          Energy per atom = -3.915555525 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 3.308 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.054796085318014   0.048530214885026  -0.114244941664615
  -0.054839988766213   0.048558286877772  -0.114316424463119
  -0.054817859762914   0.048537585831773  -0.114290282586182
  -0.054815171023411   0.048545290721319  -0.114290296476483
   0.054813423831444  -0.048563023248182   0.114239909776553
   0.054834877872459  -0.048531974330337   0.114237986270811
   0.054808945487214  -0.048535984675891   0.114333488981828
   0.054811857679436  -0.048540396061480   0.114330560161207

 Max magnitude of forces (Ha/Bohr):   0.135770307852691


 Time for calculating forces: 3.646386 s.

["Stress in GPa"


] -16.458989112899328  -8.362634519186942  -3.885419689701377
  -8.362634519186942 -22.523365016189899   7.046333313846787
  -3.885419689701377   7.046333313846787 -14.641892359686834


 Time for calculating stress: 8.014899 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  0.000000 	  0.000000 	  0.000000 
  5.100000 	  5.100000 	  0.000000 
  0.000000 	  5.100000 	  5.100000 
  5.100000 	  0.000000 	  5.100000 
  2.550000 	  2.550000 	  2.550000 
  7.650000 	  7.650000 	  2.550000 
  2.550000 	  7.650000 	  7.650000 
  7.650000 	  2.550000 	  7.650000 

 Run-time of the program: 54.374638 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.935829926919971
                        L1: 10.199999999999999
                        L2: 10.199999999999999
                        L3: 10.199999999999999
                        Nx: 25
                        Ny: 25
                        Nz: 25
                         N: 15625
                      ecut: -1
              mesh_spacing: -1
                   kptgrid: [0 0 0]
                  kptshift: [0 0 0]
                      nkpt: [1 1 1]
                     tnkpt: 1
                      wkpt: 1
                        BC: 2
                       BCx: 0
                       BCy: 0
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 30
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157730000000000e+02
                       bet: 1.000006412214051e+03
                       npl: 20
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 32
                     n_typ: 1
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 1.664640000000000e-04
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 0.010000000000000
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 1
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'Si8'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x1 struct]
                     n_atm: 8
                     Atoms: [8x3 double]
           mvAtmConstraint: [8x3 double]
                    IsFrac: 1
                    IsSpin: 0
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [8x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.408000000000000
                        dy: 0.408000000000000
                        dz: 0.408000000000000
                        dV: 0.063559053125561
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [15625x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [325x1 logical]
                 isOutr_11: [325x1 logical]
                      I_11: [325x1 double]
                     II_11: [325x1 double]
                      V_11: [325x1 double]
                 isOutl_22: [325x1 logical]
                 isOutr_22: [325x1 logical]
                      I_22: [325x1 double]
                     II_22: [325x1 double]
                      V_22: [325x1 double]
                 isOutl_33: [325x1 logical]
                 isOutr_33: [325x1 logical]
                      I_33: [325x1 double]
                     II_33: [325x1 double]
                      V_33: [325x1 double]
                  isOutl_1: [300x1 logical]
                  isOutr_1: [300x1 logical]
                       I_1: [300x1 double]
                      II_1: [300x1 double]
                       V_1: [300x1 double]
                  isOutl_2: [300x1 logical]
                  isOutr_2: [300x1 logical]
                       I_2: [300x1 double]
                      II_2: [300x1 double]
                       V_2: [300x1 double]
                  isOutl_3: [300x1 logical]
                  isOutr_3: [300x1 logical]
                       I_3: [300x1 double]
                      II_3: [300x1 double]
                       V_3: [300x1 double]
                 G_JOutl_1: [187500x1 logical]
                 G_JOutr_1: [187500x1 logical]
                     G_J_1: [187500x1 double]
                     G_I_1: [187500x1 double]
                     G_V_1: [187500x1 double]
                 G_JOutl_2: [187500x1 logical]
                 G_JOutr_2: [187500x1 logical]
                     G_J_2: [187500x1 double]
                     G_I_2: [187500x1 double]
                     G_V_2: [187500x1 double]
                 G_JOutl_3: [187500x1 logical]
                 G_JOutr_3: [187500x1 logical]
                     G_J_3: [187500x1 double]
                     G_I_3: [187500x1 double]
                     G_V_3: [187500x1 double]
                   Lap_std: [15625x15625 double]
                    grad_1: [15625x15625 double]
                    grad_2: [15625x15625 double]
                    grad_3: [15625x15625 double]
                LapPreconL: [15625x15625 double]
                LapPreconU: [15625x15625 double]
              memory_usage: 60750008
               staticfname: 'Si8.static'
                 suffixNum: 0
                  outfname: 'Si8.out'
                  parallel: 0
                         b: [15625x1 double]
                     b_ref: [15625x1 double]
                    rho_at: [15625x1 double]
                     Eself: 1.021944589659063e+02
                 Eself_ref: 2.254100666281729e+02
                       V_c: [15625x1 double]
                 PosCharge: 32.000000021287491
                 NegCharge: -32.000000021287491
                    E_corr: -2.299183892944257e-04
                       rho: [15625x1 double]
                      Atom: [1x8 struct]
                       phi: [15625x1 double]
                      e_xc: [15625x1 double]
                 dvxcdgrho: [15625x1 double]
                       Vxc: [15625x1 double]
                      Veff: [15625x1 double]
                         X: [15625x7 double]
                         F: [15625x7 double]
          mixing_hist_fkm1: [15625x1 double]
          mixing_hist_xkm1: [15625x1 double]
                       psi: [15625x30 double]
    upper_bound_guess_vecs: [15625x1 double]
                    EigVal: [30x1 double]
                  lambda_f: 0.233558136752292
                       occ: [30x1 double]
                    Etotal: -31.324444198695957
                     Eband: 1.372394486970589
                       Exc: -9.776403227640136
                    Exc_dc: -12.723942516023010
                  Eelec_dc: 66.550518035784648
                      Eent: -2.084522903247567e-04
                      Escc: 1.326751827927992e-06
                   S_Debug: [1x1 struct]
                     force: [8x3 double]
                 abs_force: [8x1 double]
                    Stress: [3x3 double]
                  Pressure: 17.874748829592022
             stress_dimred: [3x3 double]

