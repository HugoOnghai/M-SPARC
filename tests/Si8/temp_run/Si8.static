Atomic forces (Ha/Bohr):
     -0.0547960853       0.0485302149      -0.1142449417
     -0.0548399888       0.0485582869      -0.1143164245
     -0.0548178598       0.0485375858      -0.1142902826
     -0.0548151710       0.0485452907      -0.1142902965
      0.0548134238      -0.0485630232       0.1142399098
      0.0548348779      -0.0485319743       0.1142379863
      0.0548089455      -0.0485359847       0.1143334890
      0.0548118577      -0.0485403961       0.1143305602
Stress (GPa):
 -1.6458989113E+01  -8.3626345192E+00  -3.8854196897E+00 
 -8.3626345192E+00  -2.2523365016E+01   7.0463333138E+00 
 -3.8854196897E+00   7.0463333138E+00  -1.4641892360E+01 
