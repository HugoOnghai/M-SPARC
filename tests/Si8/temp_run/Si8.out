***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:09:15                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.200000 10.200000 10.200000 
LATVEC:
1.000000000000000 0.000000000000000 0.000000000000000 
-0.292371704722737 0.956304755963035 0.000000000000000 
0.173648177666930 -0.110492654830881 0.978589640054184 
FD_GRID: 25 25 25
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999994
CHEB_DEGREE: 20
NSTATES: 30
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si8.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.408000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.75 7.75 7.75
Number of atoms of type 1          :  8
Estimated total memory usage       :  57.94 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9136362203E+00        2.309E-02        8.587
2            -3.9152263564E+00        1.431E-02        2.152
3            -3.9154557077E+00        8.513E-03        2.220
4            -3.9155372649E+00        4.217E-03        2.176
5            -3.9155511567E+00        2.187E-03        2.032
6            -3.9155548679E+00        8.991E-04        1.962
7            -3.9155553450E+00        4.390E-04        1.935
8            -3.9155554884E+00        1.894E-04        1.956
9            -3.9155555406E+00        6.915E-05        1.849
10           -3.9155555403E+00        2.569E-05        1.782
11           -3.9155555289E+00        1.375E-05        1.798
12           -3.9155555276E+00        4.313E-06        1.805
13           -3.9155555269E+00        2.035E-06        1.648
14           -3.9155555248E+00        7.966E-07        1.631
Total number of SCF: 14    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.9155555248E+00 (Ha/atom)
Total free energy                  : -3.1324444199E+01 (Ha)
Band structure energy              :  1.3723944870E+00 (Ha)
Exchange correlation energy        : -9.7764032276E+00 (Ha)
Self and correction energy         : -1.0219468888E+02 (Ha)
Entropy*kb*T                       : -2.0845229032E-04 (Ha)
Fermi level                        :  2.3355813675E-01 (Ha)
Average force                      :  1.3572956395E-01 (Ha/Bohr)
Maximum force                      :  1.3577030785E-01 (Ha/Bohr)
Time for force calculation         :  3.646 (sec)
Pressure                           :  1.7874748830E+01 (GPa)
Max stress                         :  2.2523365016E+01 (GPa)
Time for stress calculation        :  8.017 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  54.375 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
